cloudify_rest_client/execution_schedules.py

```diff
@@ -134,25 +134,32 @@ def create(self, schedule_id, deployment_id, workflow_id,
             the scheduler won't make further attempts to run it.
         :param created_by: Override the creator. Internal use only.
         :param created_at: Override the creation timestamp. Internal use only.
+        :param enabled: Boolean indicating whether the schedule should be
+                        enabled.
         :return: The created execution schedule.
         """
         assert schedule_id
         assert deployment_id
         assert workflow_id
         assert since
+        if hasattr(since, 'isoformat'):
+            since = since.isoformat()
+        if hasattr(until, 'isoformat'):
+            until = until.isoformat()
```

Comment:
wouldn't you prefer isinstance(datetime)?

Reply:
Could go either way. I'll switch it.